src/extension/extensionServer.ts

```diff
@@ -249,6 +252,13 @@ export class ExtensionServer implements vscode.Disposable {
                     })
                     .catch(error => {
                         this.logger.error(error);
+                        if (error instanceof InternalError && error.errorCode === InternalErrorCode.ReactNativeDependencyIsNotInstalled) {
+                            this.logger.warning("It seems that 'react-native' package is not installed. Please run 'npm install' to install the package.");
```

Comment:
Does this string need to be localized?

Reply:
Yes, It does. We added localization for this error message.